docs_overrides/main.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+
+{% block content %}
+{% if page.edit_url %}
+  <a href="{{ page.edit_url
+    | replace('/amundsen/edit/master/docs/metadata/docs/','/amundsenmetadatalibrary/edit/master/docs/')
+    | replace('/amundsen/edit/master/docs/common/docs/','/amundsencommon/edit/master/docs/')
+    | replace('/amundsen/edit/master/docs/databuilder/docs/','/amundsendatabuilder/edit/master/docs/')
+    | replace('/amundsen/edit/master/docs/frontend/docs/','/amundsenfrontendlibrary/edit/master/docs/')
+    | replace('/amundsen/edit/master/docs/search/docs/','/amundsensearchlibrary/edit/master/docs/')
+    | replace('/amundsen/edit/master/docs/metadata/docs/','/amundsenmetadatalibrary/edit/master/docs/')
+    | replace('/amundsen/edit/master/docs/CONTRIBUTING.md','/amundsen/edit/master/CONTRIBUTING.md')
+    | replace('/amundsen/edit/master/docs/LICENSE','/amundsen/edit/master/LICENSE')
+    | replace('/amundsen/edit/master/docs/index.md','/amundsen/edit/master/README.md')
+    | replace('/amundsen/edit/master/docs/k8s_install.md','/amundsen/edit/master/amundsen-kube-helm/README.md')
+     }}" title="{{ lang.t('edit.link.title') }}" class="md-content__button md-icon">
+    {% include ".icons/material/pencil.svg" %}
```

Comment:
@sbrugman  could you help to take a look at the CI error as this svg is missing?

Reply:
Will fix it. Mismatch in mkdocs-material version.